src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/CreateResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,141 @@
+@ignore
+Feature: Resources "Create" Operation validations
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+
+        @ignore
+        Scenario: Verify new resource can be created successfully
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave",
+                        "effectiveBeginDate": "2024-05-14",
+                        "effectiveEndDate": "2024-05-14",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 201
+              And the record can be retrieved with a GET request
+                            # TODO: when writing the step definition,
+                            # be sure to validate that the response matches the original payload,
+                            # except for the addition of the 'id' field
+
+        @ignore
+        Scenario: Verify error handling with POST using invalid data
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "",
+                        "description": "Wrong Value",
+                        "effectiveBeginDate": "2024-05-14",
+                        "effectiveEndDate": "2024-05-14",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Wrong Value"
+                    }
+                  """
+             Then it should respond with 400
+              And the response body is
+                  """
+                    {
+                        "detail": "Data validation failed. See 'validationErrors' for details.",
+                        "type": "urn:ed-fi:api:bad-request:data",
+                        "title": "Data Validation Failed",
+                        "status": 400,
+                        "correlationId": null,
+                        "validationErrors": {
+                        "$.codeValue": [
+                            "CodeValue is required and should not be left empty."
+                        ]
+                  }
+                    }
+                  """
+
+        @ignore
+        Scenario: Verify error handling with POST using invalid data Forbidden
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "",
+                        "description": "Wrong Value",
+                        "namespace": "uri://.org/wrong",
```

Comment:
I'm glad you discovered this interesting scenario.

Reply:
@stephenfuqua do you want me to report an issue?

Reply:
This is the correct error output. I just forgot about the scenario, which is unique to Descriptors. So for a moment I was surprised by this outcome, but it is exactly what we need.